src/main/java/seedu/meditracker/medication/MedicationList.java

```diff
@@ -0,0 +1,26 @@
+package seedu.meditracker.medication;
+
+import java.util.ArrayList;
+
+/**
+ * The MedicationList class represents a list of medications.
+ * It contains an ArrayList of Medication objects.
+ */
+public class MedicationList {
+    /** The list of medications stored in an ArrayList. */
+    public ArrayList<Medication> medications = new ArrayList<Medication>();
```

Comment:
Might be better to use type inference and also use interface instead of explicit type of list

```suggestion
    public List<Medication> medications = new ArrayList<>();
```